dali/operators/reader/numpy_reader_op.cc

```diff
@@ -49,21 +49,21 @@ void NumpyReader::TransposeHelper(Tensor<CPUBackend>& output, const Tensor<CPUBa
 DALI_REGISTER_OPERATOR(NumpyReader, NumpyReader, CPU);
 
 DALI_SCHEMA(NumpyReader)
-  .DocStr("Read Numpy arrays from a directory")
+  .DocStr("Reads the Numpy arrays from a directory.")
   .NumInput(0)
   .NumOutput(1)  // (Arrays)
   .AddArg("file_root",
-      R"code(Path to a directory containing data files.
-`NumpyReader` supports flat directory structure. `file_root` directory should contain
-directories with numpy files in them.)code",
-      DALI_STRING)
+      R"code(Path to a directory that contains the data files.
```

Comment:
Why this change, I wonder? What's wrong with "containing"? I think it's both shorter and somehow more familiar. I've seen it used like this many times in various kinds of software (please enter the path to a folder containing the installation files and similar).

Reply:
Done